lib/modules/ens/embarkjs.js

```diff
@@ -298,7 +185,10 @@ __embarkENS.lookup = function(address) {
     let resolverContract = new web3.eth.Contract(self.resolverInterface, resolverAddress);
     return resolverContract.methods.name(node).call();
   }).then((name) => {
-    if (name === "" || name === undefined) throw Error("ENS name not found");
+    if (name === "" || name === undefined) {
+      console.warn("ENS name not found");
+      return;
+    }
     return name;
   }).catch(err => err);
```

Comment:
Need to log and/or callback the error

Reply:
You cna legit remove the whole thing and just keep `return self.ens.methods.resolver(node).call()`